tests/features/master.feature

```diff
@@ -7,6 +7,5 @@ Feature: CLI updater - master base
     And the version number is decreased in the config.php to enforce upgrade
     When the CLI updater is run successfully
     And the output should contain "Update successful"
-    Then the installed version should be 25.0
```

Comment:
Are you certain? ;)

Can you try `Then the installed version should be 26.0`. 
I assume the test fails because we forgot to bump the version here after branch off.

Reply:
done